colossalai/communication/utils.py

```diff
@@ -107,3 +107,34 @@ def gather_split_1d_tensor(tensor):
     chunks = [gathered[i*numel:(i+1)*numel] for i in range(world_size)]
     dist.all_gather(chunks, tensor, group=gpc.get_group(ParallelMode.PARALLEL_1D))
     return gathered
+
+import os
+import torch
+import warnings
+def check_single_machine_multi_gpu_p2p_available(gpu_list):
+    unavailble = []
+    check_p2p_result = os.popen("./simpleP2P")
+    p2p_info = check_p2p_result.read().split('\n')
+    for index in range(len(p2p_info)):
+        if "No" in p2p_info[index]:
+            unavailble.append(p2p_info[index])
+    print(unavailble)
```

Comment:
do not use print. And this line seems used for debugging.

Reply:
ok